### DISCUSSION TITLE
Auth Token

### DISCUSSION THREAD

**Discussion by bitehandle** [victornpb/undiscord#467] [4. Bugs and Issues]
Could not detect the authorization token!
Please make sure Undiscord is up to date

error not sure how to enter manually 
on ver 5.2.0

**Comment by psjnek** [comment]
> Could not detect the authorization token! Please make sure Undiscord is up to date
> 
> error not sure how to enter manually on ver 5.2.0

I have had the same issue. So is there maybe a way to find the token otherwise?
